src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -244,6 +270,29 @@ components:
         - status
         - sequenceNumber
 
+    CreateActionPlanRequest:
+      title: CreateActionPlanRequest
+      type: object
+      description: A request to create an  Action Plan with at least one or more Goals that a Prisoner aims to complete.
+      properties:
+        prisonNumber:
```

Comment:
Not that I mind one way or the other, but do we need prison number in the request body? Its in the URI path .... 🤔

Reply:
I've removed it from the request body 👍